packages/client/src/private-functions/executeTransactions.ts

```diff
@@ -179,17 +179,26 @@ export const executeTransactions = async (
         };
       } else if ("svmTx" in tx) {
         await validateEnabledChainIds(tx.svmTx?.chainId ?? "");
-        txResult = await executeSvmTransaction(tx, options);
+        txResult = await executeSvmTransaction(tx, options, i);
       } else {
         throw new Error("executeRoute error: invalid message type");
       }
     }
 
     await onTransactionBroadcast?.({ ...txResult });
 
+    if (txResult.explorerLink) {
+      options.onTransactionTracked?.({
+        txHash: txResult.txHash,
+        chainId: txResult.chainId,
+        explorerLink: txResult.explorerLink ?? "",
+      });
+    }
+
```

Comment:
if `txResult.explorerLink` exists, that means we used the `/submit` endpoint, so we can treat it as tracked